src/foundation.cs

```diff
@@ -13738,4 +13740,57 @@ interface NSUnitVolume : NSSecureCoding {
 		[Export ("metricCups", ArgumentSemantic.Copy)]
 		NSUnitVolume MetricCups { get; }
 	}
+
+	[iOS (10,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface NSMeasurement<UnitType> : NSCopying, NSSecureCoding 
+		where UnitType : NSUnit {
+
+		[Export ("unit", ArgumentSemantic.Copy)]
+		NSUnit Unit { get; }
+
+		[Export ("doubleValue")]
+		double DoubleValue { get; }
+
+		[Export ("initWithDoubleValue:unit:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (double doubleValue, NSUnit unit);
+
+		[Export ("canBeConvertedToUnit:")]
+		bool CanBeConvertedToUnit (NSUnit unit);
```

Comment:
Maybe `CanBeConvertedTo` instead? Otherwise it ends up sounding somewhat incomplete.